examples/asr/speech_to_text_infer.py

```diff
@@ -100,7 +100,7 @@ def main():
             reference = ''.join([labels_map[c] for c in seq_ids[0:seq_len]])
             references.append(reference)
         del test_batch
-
+    print("Ground Truth", references, "Predicted", hypotheses)
```

Comment:
This can be reverted, no need for batch of samples to be shown.

Reply:
Sorry. Just removed it.